docs/content/integration/openid-connect/jellyfin/index.md

```
@@ -21,9 +21,9 @@ seo:
 ## Tested Versions
 
 * [Authelia]
-    * [v4.38.6](https://github.com/authelia/authelia/releases/tag/v4.38.6)
+    * [v4.38.17](https://github.com/authelia/authelia/releases/tag/v4.38.17)
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)